api/test_account_deletion.sh

```
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# Simple test script for account deletion functionality
+# This script tests the DELETE /api/v1/users/me endpoint
+
+set -e
+
+API_BASE_URL="http://localhost:8080"
+AUTH_TOKEN=""
```

Shellcheck (0.10.0)

[warning] AUTH_TOKEN appears unused. Verify use (or export if used externally).

(SC2034)